tsdb/mapper.go

```diff
@@ -216,7 +216,7 @@ func (lm *LocalMapper) Open() error {
 					// No data exists for this key.
 					continue
 				}
-				seriesTags := lm.shard.index.series[key].Tags
```

Comment:
Exactly, that is what I wanted to confirm.

On Thursday, August 20, 2015, Paul Dix <notifications@github.com> wrote:

> In tsdb/mapper.go
> <https://github.com/influxdb/influxdb/pull/3749#discussion_r37539106>:
>
> > @@ -216,7 +216,7 @@ func (lm *LocalMapper) Open() error {
> >  					// No data exists for this key.
> >  					continue
> >  				}
> > -				seriesTags := lm.shard.index.series[key].Tags
>
> The nil check is just standard because we don't know who is calling the
> function. If it's part of the exported API, it should be safe.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/influxdb/influxdb/pull/3749/files#r37539106>.
>